scripts/rook.sh

```diff
@@ -26,7 +26,15 @@ function deploy_rook() {
 		kubectl create -f "${TEMP_DIR}/cluster-test.yaml"
 		rm -rf "${TEMP_DIR}"
 	else
-		kubectl create -f "${ROOK_URL}/cluster-test.yaml"
+		# add "mon_warn_on_pool_no_redundancy = false" to ceph.conf if missing
+		# see https://github.com/rook/rook/pull/5925 for upstream status
+		TEMP_DIR="$(mktemp -d)"
+		curl -o "${TEMP_DIR}"/cluster-test.yaml "${ROOK_URL}/cluster-test.yaml"
```

Comment:
Can we curl directly in "/tmp" so that we won't need to delete it to? (As it's automatically cleaned up)

Reply:
I'm not a big fan of creating the directory, but that is the pattern used here and in other scripts.
Also, cleaning up temporary files after use is recommended, you dont want to clutter /tmp with lots of unneeded stuff (it is also a RAM backed filesystem, so unneeded space has a cost).

Reply:
Okay :+1: